cmd/root.go

```diff
@@ -81,7 +81,7 @@ var RootCmd = &cobra.Command{
 
 		// Etcd is only valid for CIS 1.5 and later,
 		// this a gatekeeper for previous versions.
-		if isEtcd() && validTargets(benchmarkVersion, []string{string(check.ETCD)}) {
+		if validTargets(benchmarkVersion, []string{string(check.ETCD)}) && isEtcd() {
```

Comment:
Does the order matter?

Reply:
Yes,  CIS 1.4 and earlier do not include `etcd.yaml`, so checking to see if it's provided by the CIS benchmark before trying to check if the host is running **Etcd** avoids misleading errors.